assets/runner_cell/src/App.js

```diff
@@ -0,0 +1,339 @@
+import React, { useEffect, useRef, useState } from "react";
+import { RiQuestionnaireLine, RiArrowDownSLine } from "@remixicon/react";
+import classNames from "classnames";
+
+const FLY_CPU_KIND_OPTIONS = ["shared", "performance"].map((kind) => ({
+  value: kind,
+  label: kind,
+}));
+
+const FLY_GPU_KIND_OPTIONS = [{ value: "", label: "None" }].concat(
+  ["a10", "a100-pcie-40gb", "a100-sxm4-80gb", "l40s"].map((kind) => ({
+    value: kind,
+    label: kind,
+  })),
+);
+
+export default function App({ ctx, payload }) {
+  const [fields, setFields] = useState(payload.fields);
+  const [showHelpBox, setShowHelpBox] = useState(false);
+
+  useEffect(() => {
+    ctx.handleEvent("update", ({ fields }) => {
+      setFields((currentFields) => ({ ...currentFields, ...fields }));
+    });
+  }, []);
+
+  function pushUpdate(field, value) {
+    ctx.pushEvent("update_field", { field, value });
+  }
+
+  function handleChange(event, push = true) {
+    const field = event.target.name;
+
+    const value =
+      event.target.type === "checkbox"
+        ? event.target.checked
+        : event.target.value;
+
+    setFields({ ...fields, [field]: value });
+
+    if (push) {
+      pushUpdate(field, value);
+    }
+  }
+
+  function handleBlur(event) {
+    const field = event.target.name;
+
+    pushUpdate(field, fields[field]);
+  }
+
+  return (
+    <div className="flex flex-col gap-4 font-sans">
+      <div className="rounded-lg border border-gray-300 bg-[#fefefe]">
+        <Header>
+          <FieldWrapper>
+            <InlineLabel label="Start runner pool" />
```

Comment:
```suggestion
            <InlineLabel label="Start FLAME pool" />
```

Or just "Start FLAME".